x-pack/test/apm_api_integration/common/utils/round_number.ts

```diff
@@ -4,36 +4,13 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { round } from 'lodash';
-
 /**
  * To avoid floating precision problems, rounds any number value to max 5 decimals
  * e.g.:
  * element = {value: 0.38770000000000004}
- * return = {value: 0.3877}
+ * return = {value: 0.387700000000000}
  * @param element
  */
-export function roundNumbers(element: any): any {
-  if (element === null || element === undefined) {
-    return element;
-  }
-
-  if (typeof element === 'number') {
-    return round(element, 5);
-  }
-
-  if (Array.isArray(element)) {
-    return element.map(roundNumbers);
-  }
-
-  if (typeof element === 'object') {
-    const acc: Record<string, unknown> = {};
-    Object.keys(element).forEach((key) => {
-      acc[key] = roundNumbers(element[key]);
-    });
-
-    return acc;
-  }
-
-  return element;
+export function roundNumber(element: number): any {
+  return parseFloat(element.toPrecision(15));
```

Comment:
Should be 5, right?
```suggestion
  return parseFloat(element.toPrecision(5));
```

Reply:
We talked and agreed that 15 will be better. https://stackoverflow.com/questions/1458633/how-to-deal-with-floating-point-number-precision-in-javascript/3644302#3644302

Also because `precision(5)` might not be ideal:
```js
Array [
 -   138162752,
 -   147386368,
 +   138160000,
 +   147390000,
   ]
```

Reply:
Ah, you've changed this from 5 decimals to 15 significant digits? If so update the PR title to reflect this change